### DISCUSSION TITLE
How to deal with records that do not have abstracts?

### DISCUSSION THREAD

**Discussion by Tsklopper** [asreview/asreview#553] [Q&A] [answered]
Some of the abstracts of the records I have been screening are missing. How could I deal with these empty records?

**Comment by MartijnUX** [comment]
Would you like to skip these empty records in your review? Is this something that you want to decide before you start screening, to skip these records? Interesting user question. Anyone?

**Comment by SagevdBrand** [comment, marked as answer]
It is indeed very important for the performance of ASReview to have as little missing data as possible. So reducing the number of missing abstracts is key! There is a [blogpost](https://asreview.nl/the-importance-of-abstracts/) on how the absence of abstracts impacts your review and how you can retrieve missing abstracts. If you want to screen books, you can use the summary of the book as an abstract. Hope this helps!
